src/components/FiatOnrampModal/index.tsx

```diff
@@ -98,7 +101,7 @@ export default function FiatOnrampModal() {
           theme: isDarkMode ? 'dark' : 'light',
           colorCode: theme.accentAction,
           defaultCurrencyCode: 'eth',
-          redirectUrl: 'https://app.uniswap.org/#/swap',
+          redirectUrl: swapUrl,
```

Comment:
swapUrl needs to be added to the deps array. Why can't the path be used without the full url?

Reply:
Added, although it's not required (it won't change throughout the lifetime of the app)